cmd/helpers/mssql/mssql.go

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package mssql
+
+import (
+	"fmt"
+	"github.com/microsoft/go-sqlcmd/cmd/sqlconfig"
+	"github.com/microsoft/go-sqlcmd/pkg/sqlcmd"
+	"os"
+)
+
+func Connect(endpoint sqlconfig.Endpoint, user sqlconfig.User, console sqlcmd.Console) *sqlcmd.Sqlcmd {
+	// BUG(stuartpa): Check what the value of fromEnvironment should be here
+	v := sqlcmd.InitializeVariables(false)
+	s := sqlcmd.New(console, "", v)
+	s.Format = sqlcmd.NewSQLCmdDefaultFormatter(false)
```

Comment:
>ewSQLCmdDefaultFormatter(false) [](http://example.com/codeflow?start=20&length=31)

opportunity here to provide a JSON formatter!

Reply:
(is there a good standard for encoding arbitrary data tables as json?)

Reply:
We (SQL Server) built "FOR XML", and then "FOR JSON" clauses (for SELECT statement) that encode result sets into XML/JSON, I presume we should clone that functionality here

Reply:
Added issue:

[https://github.com/microsoft/go-sqlcmd/issues/151](https://github.com/microsoft/go-sqlcmd/issues/151)